dockerfiles/machine-exec/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM centos:7
+
+ENV DEP_DOWNLOAD_URL https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64
+ENV GOLANG_VERSION 1.10
+ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
+ENV GOLANG_DOWNLOAD_SHA256 b5a64335f1490277b585832d1f6c7f8c6c11206cba5cd3f771dcb87b98ad1a33
+# Go workspace path
+ENV GOPATH /go
+# Go installation path
+ENV PATH ${GOPATH}/bin:/usr/local/go/bin:${PATH}
+ENV MACHINE_EXEC_HASH b57b40eb2088271dc24f538f224f8c1ac474a777
+ENV MACHINE_EXEC github.com/eclipse/che-theia-terminal-plugin/machine-exec-server
+
+RUN yum -y update && yum install -y git && git --version && \
```

Comment:
" git --version "
Do we really need it?

Reply:
Nice catch, I will fix it.